src/devtools/client/webconsole/webconsole-ui.js

```diff
@@ -361,8 +361,8 @@ class WebConsoleUI {
     }
   }
 
-  _onChangeSplitConsoleState() {
-    this.wrapper.dispatchSplitConsoleCloseButtonToggle();
+  _onChangeSplitConsoleState(selectedPanel) {
```

Comment:
`selectedPanel ` => `shouldDisplayButton`

Reply:
`selectedPanel` is correct here since this is where it's being called:

https://github.com/RecordReplay/devtools/blob/f030aa2561aea1e082f2167a759dc51c4f5e8bf2/src/ui/components/Toolbox.js#L111

So it's value is the panel e.g. `debugger`, `console`, etc